libdnf/nsvcap.cpp

```diff
@@ -18,16 +18,20 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#define HEXADECIMAL "0-9abcdef"
+
 #include "nsvcap.hpp"
 
+#include "libdnf/utils/utils.hpp"
+
 #include <regex/regex.hpp>
 
 namespace libdnf {
 
-#define MODULE_NAME "([-a-zA-Z0-9\\._]+)"
+#define MODULE_NAME "([^:(/=<> ]+)"
```

Comment:
Can you use positive regex?
Something like:
`#define MODULE_NAME "([][+*?._" ASCII_LETTERS DIGITS "-]+)"`